### ISSUE TITLE
Rebuild/duplicate lp-dashboard table in website

### ISSUE THREAD

**Issue by ciaran-** [vegaprotocol/frontend-monorepo#2793]
Depends on:  https://github.com/vegaprotocol/frontend-monorepo/issues/3302

## Tasks
- [x] Get data in for MVP
- [x] Make it look like the designs as much as we can in the table 
- [x] Deploy 
- [ ] Get data in for Next Up
- [ ] Repeat

## Where it goes 

![Image](https://user-images.githubusercontent.com/46783148/232729139-2ad8b446-c66c-49b3-a938-e6293331ebd8.png)

## Design 

https://www.figma.com/file/ZQrh7fiFciTQ9GWD4Pp1Am/LP-Dashboard?type=design&node-id=1536-13160&t=F5M7U5at2xKoNuJz-0

## The data
MVP

- [x] Market Name
- [x] Price
- [x] Volume
- [x] Target Stake
- [x] Liquidity Supplied & Percentage
- [x] Liquidity Fee
- [x] Settlement Asset

Next Up

- [ ] Change in volume
- [ ] Health Bar
- [ ] Age
- [ ] Expiry
- [ ] Change in price
- [ ] Market Code

**Comment by ciaran-** [comment]
@ckpringle not sure what criteria for this looks like really, yet.

**Comment by JonRay15** [comment]
Will move the "Next Up" to another ticket

**Comment by JonRay15** [comment]
Next Up stuff is in a new ticket
